packages/strapi-admin/admin/src/translations/en.json

```diff
@@ -41,7 +41,7 @@
   "app.components.HomePage.support.link": "GET YOUR T-SHIRT NOW",
   "app.components.HomePage.welcome": "Welcome on board!",
   "app.components.HomePage.welcome.again": "Welcome ",
-  "app.components.HomePage.welcomeBlock.content": "We are happy to have you as one of community member. We are constantly looking for feedback so feel free to send us DM on ",
+  "app.components.HomePage.welcomeBlock.content": "We are happy to have you as one of the community. We are constantly looking for feedback so feel free to send us DM on ",
```

Comment:
I would say "as one of the community members" but not sure about that. What do you think @derrickmehaffy ?

Reply:
Agreed with your suggestion @alexandrebodin 

`We are happy to have you as one of community members.`

Reply:
Alternative would be `We are happy to have you as part of the community`

Reply:
Agree with @derrickmehaffy here, have updated the PR with that copy.